Client/Frontend/Browser/MainMenuActionHelper.swift

```diff
@@ -629,7 +629,7 @@ class MainMenuActionHelper: PhotonActionSheetProtocol, FeatureFlaggable, CanRemo
     private func getBookmarkLibraryAction() -> SingleActionViewModel {
         return SingleActionViewModel(title: .AppMenu.Bookmarks,
                                      iconString: ImageIdentifiers.bookmarks,
-                                     isNew: User.shared.showsBookmarksNewBadge) { _ in
+                                     isNew: EcosiaInstallType.get() == .upgrade && User.shared.showsBookmarksNewBadge) { _ in
```

Comment:
From what I've seen, `User.shared.hideBookmarksNewBadge()` is only called when the user visits bookmarks for the first time. This means that, once the user updates the app for the first time after installing, independent of the version, it would then start showing the badge until bookmarks is opened (and if it was not opened before). Does that make sense?

I believe this is minor and could be handled separately since not part of the original problem, but I think it could be easily fixed by adding `User.shared.hideBookmarksNewBadge()` inside `BrowserViewController.handleFirstTimeUserActions()` so that it is hidden for new users.

Might even also apply to `User.shared.showsBookmarksImportExportTooltip` 🤔

Reply:
Thanks for the input @lucaschifino!
If I get it correctly, the issue you are pointing at covers the scenario when a user has not opened the Bookmark section (yet and upgrades). If I got it correctly, that makes sense and the `New` badge will appear. Is your suggestion to add `User.shared.hideBookmarksNewBadge()` as well as keeping the current changes in this PR?
If so, adding both 👇  to `BrowserViewController.handleFirstTimeUserActions()` would make it work?
```swift
        User.shared.hideBookmarksNewBadge()
        User.shared.hideBookmarksImportExportTooltip()
```

Reply:
Exactly! It would basically guarantee that a new user never sees the badge nor the tooltip, if that is what we want. Do you think that is what we want? Not sure if something like that appeared in your discussions with Yann.

Reply:
What appeared from the discussion that Yann and I had was essentially the following:

> A new user should never see the tooltip or the badge.

All the considerations about what _new_ means still stand, but that's the overall desire, design-wise.
So I guess that your point is valid.